index.bs

```diff
@@ -2116,7 +2118,9 @@ When verifying a given {{PublicKeyCredential}} structure (|credential|) as part
 10. Using the credential public key looked up in step 1, verify that |sig| is a valid signature over the binary concatenation of
     |aData| and |hash|.
 
-11. If all the above steps are successful, continue with the authentication ceremony as appropriate. Otherwise, fail the
+11. Verify that the signature counter value is either (a) greater than the value stored or (b) it is equal to the value stored and the value stored is 0. If the signature counter violates this condition, this is an indication that the authenticator was cloned, i.e. at least two copies of the credential private key exist and are used in parallel.  This is a signal of potential fraud.
```

Comment:
I find this proposal even more confusing.  Since the "If this condition is not true" clause doesn't work here - it still depends on the both value are 0 one.

Here is the rule in logic notation
if (  (assertion.signCounter > db.signCounter)
     OR((assertion.signCounter==0)AND(db.signCounter==0)) )
   then everyting is fine;
   else we have counter mismatch.

More english language makes it harder to understand.  I would propose plain logic.

Reply:
Have done the following:
11. If the [=signature counter=] value |adata|.|signCount| is nonzero or the value stored 
    in conjunction with |credential|'s {{Credential/id}} attribute
    is nonzero, then set |signatureCounterSupported| to true.

12. If |signatureCounterSupported| is true 
        If the [=signature counter=] value |adata|.|signCount| is
          <dl class="switch">
              <dt>greater than the [=signature counter=] value stored in conjunction 
                  with |credential|'s {{Credential/id}} attribute.</dt>
              <dd>Update the stored [=signature counter=] value, associated with 
                  |credential|'s {{Credential/id}} attribute, to be the value of 
                  |adata|.|signCount|.</dd>
	      <dt>less than or equal to the [=signature counter=] value stored in conjunction 
                  with |credential|'s {{Credential/id}} attribute.</dt>
              <dd>This is an signal that 
                  the authenticator may be cloned, i.e. at least 
                  two copies of the [=credential private key=] may exist and are  
                  being used in parallel. [=[RPS]=] should incorporate this information  
                  into their risk scoring.  Whether the [=RP=] updates the 
                  stored [=signature counter=] value in this case, or not, or fails the 
                  authentication ceremony or not, is 
                  [=RP=]-specific. </dd>
      </dl>